cmd/snaptel/flags.go

```diff
@@ -49,10 +49,11 @@ var (
 		Name:  "running",
 		Usage: "Shows running plugins",
 	}
-	flPassword = cli.BoolFlag{
```

Comment:
@IzabellaRaulin, the reason changing to the string flag instead of bool has two benefits:
* Supports environmental variable. It does not work totally for the current implementation which uses `bool`
* Original implementation was not user-friendly. When you specify a flag `-p`, intuition is to follow a password. It does not work by using `bool`. It's quite confusing. Please try it out yourself.

The change is good! Please feel free to share your suggestions.